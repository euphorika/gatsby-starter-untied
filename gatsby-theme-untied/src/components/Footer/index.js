/** @jsx jsx */
import { jsx } from 'theme-ui'

const Footer = () => (
  <footer
    sx={{
      display: ['block', 'inline-flex'],
      justifyContent: 'space-between',
      width: '100%',
      textAlign: 'center',
      borderTop: '1px solid',
      borderColor: 'primary',
      bg: 'secondary',
      p: '1rem 1.95rem',
      color: 'primary',
    }}
  >
    <div
      sx={{
        fontSize: 3,
        mb: '10px',
      }}
    >
      Gatsby Theme Untied
    </div>
    <div
      sx={{
        fontSize: 0,
      }}
    >
      &copy; 2018
    </div>
  </footer>
)

export default Footer
