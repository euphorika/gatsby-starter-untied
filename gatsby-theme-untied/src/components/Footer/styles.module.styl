@require '../../themes/standard/config/breakpoints'
@require '../../themes/standard/config/colors'

.footer-container
  footer
    text-align: center
    background: primary-color
    padding: 1.45rem 1.0875rem
    color: secondary-color

    .brand
      font-size: 18px
      margin-bottom: 10px

    .copyright
      font-size: 12px
