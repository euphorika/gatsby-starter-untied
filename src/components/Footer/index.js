import React from 'react'

const Footer = () => (
  <footer style={{
    textAlign: 'right',
    background: 'rebeccapurple',
    padding: '1.45rem 1.0875rem',
    color: '#fff'
  }}>
    <span style={{fontSize: '12px'}}>&copy; 2018</span>
  </footer>
)

export default Footer
