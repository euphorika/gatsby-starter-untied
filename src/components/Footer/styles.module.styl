@require '../../themes/standard/config/breakpoints'
@require '../../themes/standard/config/colors'

.footer-container
  footer
    text-align right
    background primary-color
    padding 1.45rem 1.0875rem
    color secondary-color

    .copyright
      font-size 12px
