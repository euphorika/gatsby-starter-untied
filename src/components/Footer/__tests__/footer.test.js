import React from 'React'
import renderer from 'react-test-renderer'
import Footer from '../'

describe('Footer', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Footer />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
