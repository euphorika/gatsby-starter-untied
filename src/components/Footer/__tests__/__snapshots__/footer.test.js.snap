// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  style={
    Object {
      "background": "rebeccapurple",
      "color": "#fff",
      "padding": "1.45rem 1.0875rem",
      "textAlign": "right",
    }
  }
>
  <span
    style={
      Object {
        "fontSize": "12px",
      }
    }
  >
    Â© 2018
  </span>
</footer>
`;
